Library/Homebrew/cleaner.rb

```diff
@@ -93,18 +93,15 @@ def clean_dir(d)
 
       next if path.directory?
 
-      if path.extname == ".la"
-        path.unlink
-      elsif path.symlink?
-        # Skip it.
-      elsif path.basename.to_s == "perllocal.pod"
-        # Both this file & the .packlist one below are completely unnecessary
+      files_to_skip = %w[perllocal.pod .packlist]
+      if path.extname == ".la" || (!path.symlink? && files_to_skip.include?(path.basename.to_s))
```

Comment:
If the `perllocal.pod` and `.packlist` files are never symlinks this can be simplified to:

```suggestion
      if path.extname == ".la" || files_to_skip.include?(path.basename.to_s)
```

I'm not familiar enough with these files or this context to judge whether that's the case or not.